package.json

```diff
@@ -40,7 +40,7 @@
     "through2": "^0.6.3",
     "typescript": "1.8.0",
     "validator": "^3.33.0",
-    "vsts-task-lib": "^0.5.6"
+    "vsts-task-lib": "^0.6.2"
```

Comment:
I have gone through all changes in vsts-task-lib. it's LIB functionality, nothing related to tasks changed